apps/server/src/connect/dataEndpoints.test.ts

```diff
@@ -1,326 +1,140 @@
-import type { Response } from 'express'
-import { HttpResponse, http } from 'msw'
-import {
-  MX_VC_GET_ACCOUNTS_PATH,
-  MX_VC_GET_IDENTITY_PATH,
-  MX_VC_GET_TRANSACTIONS_PATH
-} from '../test/handlers'
-import {
-  mxVcAccountsData,
-  mxVcIdentityData,
-  mxVcTranscationsData
-} from '../test/testData/mxVcData'
-import { server } from '../test/testServer'
-import type {
-  AccountsRequest,
-  IdentityRequest,
-  TransactionsRequest
-} from './dataEndpoints'
+import type { Response } from "express";
+import type { TransactionsRequest } from "./dataEndpoints";
 import {
   accountsDataHandler,
   identityDataHandler,
-  transactionsDataHandler
-} from './dataEndpoints'
-import type { Aggregator } from '../shared/contract'
-import { Aggregators } from '../shared/contract'
-import { invalidAggregatorString } from '../utils/validators'
+  transactionsDataHandler,
+} from "./dataEndpoints";
+import type { Aggregator } from "../shared/contract";
+import { Aggregators } from "../shared/contract";
+import { invalidAggregatorString } from "../utils/validators";
 
 /* eslint-disable @typescript-eslint/unbound-method */
 
-describe('dataEndpoints', () => {
-  describe('accountsDataHandler', () => {
-    it('responds with a failure if aggregator isnt valid', async () => {
+describe("dataEndpoints", () => {
+  describe("accountsDataHandler", () => {
+    it("responds with a failure if aggregator isnt valid", async () => {
       const res = {
         send: jest.fn(),
-        status: jest.fn()
-      } as unknown as Response
+        status: jest.fn(),
+      } as unknown as Response;
 
       await accountsDataHandler(
         {
           params: {
-            connectionId: 'testConnectionId',
-            aggregator: 'junk' as Aggregator,
-            userId: 'testUserId'
-          }
+            connectionId: "testConnectionId",
+            aggregator: "junk" as Aggregator,
+            userId: "testUserId",
+          },
         },
-        res
-      )
-
-      expect(res.status).toHaveBeenCalledWith(400)
-      expect(res.send).toHaveBeenCalledWith(invalidAggregatorString)
-    })
-
-    it('responds with the vc data in the jwt on success', async () => {
```

Comment:
We're losing a lot of coverage with all these test deletions

Reply:
Good point. I should put these tests back in, but use the TestAdapter instead of MX.

Reply:
Okay, these should all be back, and are now using a test adapter.